services/web/server/src/simcore_service_webserver/catalog/_models.py

```diff
@@ -54,43 +53,29 @@ def get_html_formatted_unit(
 #
 # Transforms from catalog api models -> webserver api models
 #
-
-
-# Caching:  https://cachetools.readthedocs.io/en/latest/index.html#cachetools.TTLCache
-# - the least recently used items will be discarded first to make space when necessary.
+# Uses aiocache (async) instead of cachetools (sync) in order to handle concurrency better
+# SEE https://github.com/ITISFoundation/osparc-simcore/pull/6169
 #
+_SECOND = 1  # in seconds
+_MINUTE = 60 * _SECOND
```

Comment:
these should be final as well no?
also sure we don't have already these 2 constants already somewhere?

Reply:
the problem is the reference. Here is seconds and somewhere else might be e.g. microseconds. I do not think this is too harming